src/SideMenu.tsx

```diff
@@ -40,14 +45,19 @@ export const SideMenu: React.FC = (props) => {
   return (
     <div className="sidemenu" ref={ref}>
       <button className="sidemenu__btn" onClick={handleSelectorClicked}>
-        {currentSideMenuItems.length > 0 && t(currentSideMenuItems[0].children)}
+        {currentSideMenuItems.length > 0 ? t(currentSideMenuItems[0].children) : t("orders")}
       </button>
       <div className={`sidemenu__dropdown ${!isOpen ? 'sidemenu__hidden' : ''}`}>
         {sideMenuItems.map(elem => (
           <div className='sidemenu__item' key={elem.to}>
-            <Link to={elem.to} className={`sidemenu__link ${location.pathname === elem.to ? '--selected' : ''}`} onClick={handleHideMenu}>{t(elem.children)}</Link>
+            {location.pathname === `/orderdetails/${orderId}`
```

Comment:
What was the reason for this change? It's the same link and action but the only difference is the classNames used. Readability?